api/app/outcome/api.py

```diff
@@ -101,20 +110,33 @@ def post(self, event_id):
                     event_id,
                     args['user_id'],
                     status,
-                    g.current_user['id'])
+                    g.current_user['id'],
+                    reason=args['reason']
+                )
 
             outcome_repository.add(outcome)
             db.session.commit()
-
-            if (status == Status.REJECTED or status == Status.WAITLIST):  # Email will be sent with offer for accepted candidates  
-                email_user(
-                    'outcome-rejected' if status == Status.REJECTED else 'outcome-waitlist',
-                    template_parameters=dict(
-                        host=misc.get_baobab_host()
-                    ),
-                    event=event,
-                    user=user,
-                )
+            print("event_type::::::::", event.event_type)
+            print("Type of event.event_type:::::::", type(event.event_type))
```

Comment:
"print" shouldn't be in the production codebase?

 (please see comments above)